README.md

```diff
@@ -48,7 +48,7 @@ The command has the following options:
         "config": {                         // All config fields optional
           "isBlocking": true,
           "overrides":{
-
+              "startingUrl": "staging-url-override.com"
```

Comment:
maybe we want to use `example.com`
context: https://news.ycombinator.com/item?id=29623646

Reply:
Good call! Will do